### DISCUSSION TITLE
How to keep the Transition or Drawer mounted on either of the true/false states?

### DISCUSSION THREAD

**Discussion by vipindigiqt** [mantinedev/mantine#3483] [Q&A] [answered]
I'm working on a Chrome extension in which I'm using the Transition component to toggle the sidebar.

The issue is, I'm losing the state of the app like route and data when I toggle the Transition.

Is there any way to keep the component mounted, and just get that transition part only?

**Comment by rtivital** [comment, marked as answer]
This prop will be available in 6.0 release https://v6.mantine.dev/changelog/6-0-0/#keepmounted-prop

**Reply by vipindigiqt** [reply]
> This prop will be available in 6.0 release https://v6.mantine.dev/changelog/6-0-0/#keepmounted-prop

Any way to get the same behavior in v5?

I might have moved to v6 alpha, but there are a lot of breaking changes (obviously).

**Reply by rtivital** [reply]
No, it cannot be achieved in 5.x
